build.gradle

```diff
@@ -546,6 +546,27 @@ project(':iceberg-azure') {
     testImplementation libs.esotericsoftware.kryo
     testImplementation libs.testcontainers
   }
+
+  sourceSets {
+    integration {
+      java.srcDir "$projectDir/src/integration/java"
+      resources.srcDir "$projectDir/src/integration/resources"
+      compileClasspath += main.output + test.output
+      runtimeClasspath += main.output + test.output
+    }
+  }
+
+  configurations {
+    integrationImplementation.extendsFrom testImplementation
+    integrationRuntime.extendsFrom testRuntimeOnly
+  }
+
+  task integrationTest(type: Test) {
+    useJUnitPlatform()
+    testClassesDirs = sourceSets.integration.output.classesDirs
+    classpath = sourceSets.integration.runtimeClasspath
+    jvmArgs += project.property('extraJvmArgs')
+  }
```

Comment:
this also needs `check.dependsOn integrationTest` right after this line. I believe the same needs to be updated for `iceberg-s3`

Reply:
Thanks for catching this. I added the line on both `iceberg-aws` and `iceberg-azure`. I did try to run `./gradlew check` and it is triggering the tests. I cannot pass them because of the docker env issue on my local machine, I will wait for PRB to verify they run on the `check` task.